src/H5F.c

```diff
@@ -462,6 +462,49 @@ H5Fget_vfd_handle(hid_t file_id, hid_t fapl_id, void **file_handle /*out*/)
     FUNC_LEAVE_API(ret_value)
 } /* end H5Fget_vfd_handle() */
 
+/*-------------------------------------------------------------------------
+ * Function:    H5Fget_file_driver_handle
+ *
+ * Purpose:     Returns a pointer to the handle of the low-level file
+ *              driver.
+ *
+ * Return:      Success:    Non-negative
+ *              Failure:    Negative
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5Fget_file_driver_handle(hid_t file_id, void **file_handle /*out*/)
```

Comment:
Since this is native-VOL-only, the name should reflect that.

Reply:
Hi Dana! Thanks for your review. This shouldn't be tied to the native VOL -- the `H5VL_optional_args_t` and `H5VL_native_file_optional_args_t` are dead variables (sorry about that! I'll remove them in my next PR). Do you see another assumption on the implementation of that function that's only guaranteed to work with the native VOL driver?